File: Python/PB9.py
'''
After finding the output of pevious question
divide it by 2 and use this answer as an index
and find the name of the sender of mail
'''
f=open("email.txt","rt")
people_who_sent_mail=[ ]
for line in f:
    if not line.startswith('From: '): continue
    pieces = line.split()
    email = pieces[1]
    people_who_sent_mail.append(email)
#print(people_who_sent_mail)
a=len(people_who_sent_mail)
middle_one=people_who_sent_mail[int(a/2)]
list_org=middle_one.split('@')
org=list_org[1]
print("Name:",org)
f.close()

File: C++/PB9.cpp
//Divide the number of lines of the file (Answer of previous question) by 2
#include <iostream>
#include <regex>
#include <fstream>
#include <string>
#include <string_view>
#include <regex>
#include <sstream>
using namespace std;
int main() {
    string myText,req,line;
    int c=0;
    int a=0;
    ifstream emailfile("email.txt");
    if (emailfile.is_open())
    {
        cout<<"Reading From File Operation Started!"<<endl;
        while (getline(emailfile; line))
        {
            c=c+1;
            //Here Comes the value
            if (c==)
            {
                cout<<line<<endl;
            }
        }
        cout<<"The Number Of Lines In the File are:"<<endl;
        cout<<c<<endl;
        cout<<req;
        emailfile.close();
    }
    else
    {
        std::cout << "File not opened." << std::endl;
    }
    return 0;
}
